### PR TITLE
Pygame 1.9.4

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: dev-python/pygame/patches/pygame-1.9.4.patchset [hunk 1]
```diff
@@ -302,18 +302,18 @@ index c54a93a..1b9d208 100644
  src/scale_mmx64_gcc.c
```
Comment (alaviss) [inline dev-python/pygame/patches/pygame-1.9.4.patchset]: Do we still need to drop these `scale` files?

File: dev-python/pygame/python_pygame-1.9.4.recipe [hunk 2]
```diff
@@ -47,8 +49,9 @@ BUILD_REQUIRES="
 	"
```
Comment (alaviss) [inline dev-python/pygame/python_pygame-1.9.4.recipe]: There shouldn't be a newline here.

File: dev-python/pygame/python_pygame-1.9.4.recipe [hunk 3]
```diff
@@ -1,16 +1,16 @@
 SUMMARY="A popular game development module for python"
 DESCRIPTION="PyGame - python bindings to sdl and other libs that facilitate \
 game production."
-HOMEPAGE="http://www.pygame.org/"
+HOMEPAGE="https://www.pygame.org/"
 COPYRIGHT="2000-2004, 2007  Pete Shinners
```
Comment (alaviss) [inline dev-python/pygame/python_pygame-1.9.4.recipe]:
```diff
-COPYRIGHT="2000-2004, 2007  Pete Shinners
+COPYRIGHT="2000-2004, 2007 Pete Shinners
```

File: dev-python/pygame/python_pygame-1.9.4.recipe (hunk 2 above)
Comment (waddlesplash) [reply dev-python/pygame/python_pygame-1.9.4.recipe]: Huh? Yes there should, this is between PREREQUIRES and BUILD().

File: dev-python/pygame/patches/pygame-1.9.4.patchset (hunk 1 above)
Comment (waddlesplash) [reply dev-python/pygame/patches/pygame-1.9.4.patchset]: Probably.

**Comment by fbrosson** [comment]
FWIW, the default merge method (between _"Squash and merge"_ and _"Rebase and merge"_) varies depending on which merge method one has chosen for the previous merge.
Here, for instance, the _"Rebase and merge"_ method was chosen, so we got: 7de1f05091a2a9ef1a17c7d89893406708c6cffd.
My suggestion: always check the merge method if there is more than one commit in the PR.

@waddlesplash, I saw that you have switched to _"Squash and merge"_ in your next merge, and this is cool! So thanls, and sorry for writing ths little reminder :)

**Pull request merged**